tests/test_aiobaro.py

```diff
@@ -274,52 +274,89 @@ async def test_thumbnail(matrix_client):
     assert result.ok
 
 
-async def test_profile_get(matrix_client):
-    args, kwargs = [], {}
-    result = await matrix_client.profile_get(*args, **kwargs)
+@pytest.mark.asyncio
+async def test_profile_set_displayname(matrix_client):
+    await test_register(matrix_client)
+    await test_login(matrix_client)
+
+    user_id = "@test_user:baro"
+    display_name = "Test USER"
+    result = await matrix_client.profile_set_displayname(user_id, display_name)
     assert result.ok
```

Comment:
you must to assert the content response! Now don't known if response contains another user display name or not

Reply:
According to the Matrix specification, the response for this action is only "{}". We then only asserted result.ok

Reply:
ok! all right! I mistaked in confusing set/get